lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
should we generate the analogous skylight configuration so you know what ENV variables you need to provide? Maybe it could have comments on how to set things up?